library/junos_install_config

```diff
@@ -198,6 +203,7 @@ def _load_via_netconf(module):
     in_check_mode = module.check_mode
     overwrite = module.boolean(module.params['overwrite'])
     replace = module.boolean(module.params['replace'])
+    template_vars_file = module.params['template_vars_file']
```

Comment:
@lamoni As you can notice its tried to keep in sync with the variable name being used in PyEZ directly. For ex: 
overwrite = module.boolean(module.params['overwrite'])
replace = module.boolean(module.params['replace'])
Similarly can we have template_vars  in place of template_vars_file (ref https://junos-pyez.readthedocs.org/en/1.2.3/jnpr.junos.utils.html#jnpr.junos.utils.config.Config.load)